Branch-SDK/src/main/java/io/branch/referral/validators/IntegrationValidatorDialog.java

```diff
@@ -0,0 +1,99 @@
+package io.branch.referral.validators;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.Intent;
+import android.view.Window;
+import android.widget.Button;
+import android.widget.TextView;
+
+import io.branch.referral.Branch;
+import io.branch.referral.R;
+
+public class IntegrationValidatorDialog extends Dialog {
+
+    IntegrationValidatorDialogRowItem test1RowItem;
```

Comment:
Could these instead be named by their UX id name? If these ever get reorganized visually it'll be easier not having to renumber them all.

Reply:
Great point, renamed with the UX names!